### ISSUE TITLE
Empty Hair object - Properties editor - Object Data Properties - Attributes Panel - missing icons in Add Attribute menu

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#3145]
This menu could benefit from icons.

![missingicons](https://user-images.githubusercontent.com/13919702/179392182-e2c93e73-c19d-48f4-a030-9eacd30a01e0.jpg)

**Comment by ReinerBforartists** [comment]
Hm. Mission impossible. These props does not allow icons - yet. @iyadahmed , any ideas here?

![noway](https://user-images.githubusercontent.com/13919702/179410840-50f34cf1-9e5b-4af1-9a8a-ad8a095eacb7.jpg)

**Comment by iyadahmed** [comment]
ah I think they can, but honestly I don't know yet

**Comment by ReinerBforartists** [comment]
Yes, i think i've done it before too. But i can't remember how ^^

**Comment by Draise14** [comment]
I think I got it, doing a test compile.

**Comment by Draise14** [comment]
![image](https://user-images.githubusercontent.com/35357409/224811636-0bfb604b-1c7e-451e-b60d-e97ed51477b0.png)

There we go. Was similar to this code from here: 
https://github.com/Bforartists/Bforartists/issues/3313
